src/sentry/api/serializers/models/organization_member.py

```diff
@@ -192,3 +193,25 @@ def serialize(
         d = super().serialize(obj, attrs, user)
         d["projects"] = attrs.get("projects", [])
         return d
+
+
+class OrganizationMemberSCIMSerializer(Serializer):  # type: ignore
+    def __init__(self, expand: Optional[Sequence[str]] = None) -> None:
+        self.expand = expand or []
+
+    def serialize(
+        self, obj: OrganizationMember, attrs: Mapping[str, Any], user: Any, **kwargs: Any
+    ) -> MutableMapping[str, JSONData]:
+
+        d = {
+            "schemas": [SCIM_SCHEMA_USER],
+            "id": obj.id,
+            "userName": obj.get_email(),  # TODO: does this get weird with secondary emails?
+            "name": {"givenName": "N/A", "familyName": "N/A"},
+            "emails": [
+                {"primary": True, "value": obj.get_email(), "type": "work"}
+            ],  # TODO: secondary emails?
```

Comment:
going to leave secondary emails out for now, we'll see if this ends up being something admins need